Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -14,19 +14,24 @@ namespace Stratis.Bitcoin.Features.Consensus
 {
     public class StakeValidator
     {
-        // Ratio of group interval length between the last group and the first group.
-        private const int ModifierIntervalRatio = 3;
+        /// <summary>Expected (or target) block time in seconds.</summary>
+        public const int TargetSpacingSeconds = 64;
```

Comment:
ModifierIntervalRatio  has been deleted?

Reply:
well ... it was and the code compiles ...